### PR TITLE
{numlib}[dummy/dummy] cuDNN v5.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/cuDNN/cuDNN-5.1-CUDA-8.0.44.eb [hunk 1]
```diff
@@ -24,7 +24,7 @@ sources = ['%(namelower)s-8.0-linux-x64-v%(version)s.tgz']
 dependencies = [('CUDA', cuda_version)]
 
 checksums = [
-    '099ca6a9b21b35e32f3408eff2a1c06d',    # cudnn-8.0-linux-x64-v5.1.tgz
+    '406f4ac7f7ee8aa9e41304c143461a69',    # cudnn-8.0-linux-x64-v5.1.tgz, Jan 20 2017 download
```
Comment (boegel) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-5.1-CUDA-8.0.44.eb]: @verdurin hmm, why are we now getting a different checksum here?

Comment (verdurin) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-5.1-CUDA-8.0.44.eb]: It's the first time I've installed it, and that's the checksum I see with the current download, which gives the date as January 20th 2017. 

I believe the easyconfig was submitted in December 2016, so perhaps Nvidia has refreshed it without changing the version?

Comment (boegel) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-5.1-CUDA-8.0.44.eb]: urgh, indeed, redownloading now gives me the same MD5 as you, great...

I wonder why @NVIDIA would just update tarballs in place without a proper version bump?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2116.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2fb082ab2e6f4de2518601db03e66572 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**